src/main.cpp

```diff
@@ -7297,6 +7275,22 @@ bool ProcessMessages(const CChainParams& chainparams, CNode* pfrom)
     return fOk;
 }
 
+class CompareInvMempoolOrder
+{
+    CTxMemPool *mp;
+public:
+    CompareInvMempoolOrder(CTxMemPool *mempool)
+    {
+        mp = mempool;
+    }
+
+    bool operator()(std::set<uint256>::iterator a, std::set<uint256>::iterator b)
+    {
+        /* As std::make_heap produces a max-heap, we want the entries with the
+         * fewest ancestors/highest fee to sort later. */
```

Comment:
```suggestion
         * highest fee to sort later. */
```
The upstream comment doesn't reflect what we actually implement in zcashd.